bucket/joyshockmapper.json

```
@@ -0,0 +1,41 @@
+{
+    "homepage": "https://github.com/Electronicks/JoyShockMapper",
+    "description": "A tool for PC gamers to play games with DualShock 4s, JoyCons, and Pro Controllers. Gyro aiming, flick stick.",
+    "license": "MIT",
+    "version": "v3.3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/v3.3.0/JoyShockMapper_x64.zip",
+            "hash": "1B3C3B5C1B1D2B09075221AE283D171820ED5198021C8DB6417CAA6BD1994DCE",
```

Use lowercase for hashes:

```suggestion
            "hash": "1b3c3b5c1b1d2b09075221ae283d171820ed5198021c8db6417caa6bd1994dce",
```